### PR TITLE
Editorial review suggestions for Security Considerations section

### PR DESCRIPTION
**Pull request by grantnoble**
Again, mostly "verifiable" credentials, but also some grammar and punctuation, and other changes.

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/528.html" title="Last updated on Apr 5, 2019, 1:26 AM UTC (fb4376d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/528/1ad7698...grantnoble:fb4376d.html" title="Last updated on Apr 5, 2019, 1:26 AM UTC (fb4376d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4251,23 +4253,24 @@ <h3>Bundling Dependent Claims</h3>
 
       <p>
 It is considered best practice for <a>issuers</a> to atomize information in a
-<a>credential</a>, or use a signature scheme that allows for selective
-disclosure. In the case of atomization, if it is not done securely by the
-<a>issuer</a>, the <a>holder</a> might bundle together different
-<a>credentials</a> in a way that was not intended by the <a>issuer</a>.
+<a>verifiable credential</a>, or use a signature scheme that allows for
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 2]
```diff
@@ -4251,23 +4253,24 @@ <h3>Bundling Dependent Claims</h3>
...
+selective disclosure. In the case of atomization, if it is not done securely by
+the <a>issuer</a>, the <a>holder</a> might bundle together different
+<a>verifiable credentials</a> in a way that was not intended by the
```
Comment (msporny) [inline index.html]: nope, credentials

File: index.html [hunk 3]
```diff
@@ -4304,7 +4308,7 @@ <h3>Device Theft and Impersonation</h3>
         </li>
         <li>
 Enabling password, biometric, or multi-factor authentication for the
-<a>credential</a> repository.
+<a>verifiable credential</a> <a>repository</a>.
```
Comment (msporny) [inline index.html]: nope, credential. Not all credentials stored in a repository will be verifiable.

**Comment by msporny** [comment]
Merge conflicts :(

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Fixed.

**Comment by grantnoble** [comment]
Merge conflicts resolved.

**Pull request merged**